tests/software/cooler/cload/test.yml

```diff
@@ -0,0 +1,7 @@
+- name: cooler cload test_cooler_cload
+  command: nextflow run tests/software/cooler/cload -entry test_cooler_cload -c tests/config/nextflow.config
```

Comment:
Add the tests corresponding to the different `tools` (`hiclib`, `tabix`, `pairs` and `pairix`)